update-bcd broken since BCD moved to fdir

After https://github.com/mdn/browser-compat-data/commit/8ad738af221a0e85685ed20aa5fe226deaa8dcbb (https://github.com/mdn/browser-compat-data/pull/5653):

```
gsnedders@gsnedders-milk mdn-bcd-collector % time npm install && npm run update-bcd -- --help

> mdn-bcd-collector@6.0.5 prepare
> node scripts.js prepare

> node install.js
Requesting latest Firefox Nightly version from https://product-details.mozilla.org/1.0/firefox_versions.json
Firefox Nightly is already in /Volumes/gsnedders/projects/mdn/bcd/mdn-bcd-collector/node_modules/puppeteer/.local-firefox/mac-102.0a1; skipping download.

up to date, audited 897 packages in 2s

105 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm install  1.63s user 0.27s system 105% cpu 1.802 total

> mdn-bcd-collector@6.0.5 update-bcd
> node update-bcd.js "--help"

node:internal/modules/cjs/loader:939
  const err = new Error(message);
              ^

Error: Cannot find module 'fdir'
Require stack:
- /Volumes/gsnedders/projects/mdn/bcd/browser-compat-data/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:939:15)
    at Module._load (node:internal/modules/cjs/loader:780:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Volumes/gsnedders/projects/mdn/bcd/browser-compat-data/index.js:8:18)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:827:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:170:29) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Volumes/gsnedders/projects/mdn/bcd/browser-compat-data/index.js'
  ]
}

Node.js v18.2.0
```

What's perhaps more surprising to me is that this keeps happening even after `npm install 'fdir@~5.2.0'`, but clearly I don't understand npm packaging :)

--- a/package.json
+++ b/package.json
@@ -78,6 +78,7 @@
     "eslint": "8.16.0",
     "eslint-config-google": "0.14.0",
     "eslint-plugin-prefer-arrow": "1.2.3",
+    "fdir": "5.2.0",
     "json3": "3.3.3",
     "klaw": "4.0.1",
     "listr2": "4.0.5",

